dashboard/src/components/Table/TestsTable.tsx

```diff
@@ -88,13 +114,72 @@ const TestsTable = ({
     </div>
   );
 
+  const onClickFilter = useCallback(
+    (filter: TableFilter) => {
+      setSelectedFilter(filter);
+      navigate({
+        search: previousParams => {
+          return {
+            ...previousParams,
+            tableFilter: filter,
```

Comment:
each tab will need to have its own state otherwise they will lose track like they are doing right now


https://github.com/user-attachments/assets/d9369102-cfd2-4fb6-87d2-e8786c13183d